resources/js/features/user/settings/components/+root/SettingsRoot.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react';
```

Comment:
generic errors appear in F12 console
![image](https://github.com/user-attachments/assets/eb01dc23-a1f9-4c91-a54d-236b6cd9dc33)

Reply:
Struggling to repo this locally:

![Screenshot 2024-09-02 at 2 47 02 PM](https://github.com/user-attachments/assets/4628de3d-d93a-45f2-8cfc-4c7bc55d2948)

It also isn't occurring on preview:

![Screenshot 2024-09-02 at 2 47 32 PM](https://github.com/user-attachments/assets/c0f4e2ae-1f5f-47f4-9626-06abb0a6a221)

Reply:
OK, it seems I can repro if a Vite dev/prod server instance isn't running.

If you do:
```
sail npm run build
sail artisan inertia:start-ssr
```
Then visit the `/settings` route, do you still see errors?

Something must be wonky with your Vite setup locally, which may also be why things were odd in the i18n PR.


https://github.com/user-attachments/assets/24d568fc-f611-477a-9b17-22ab3d8f2924

Reply:
`sail artisan inertia:start-ssr` does indeed eliminate those console errors. Is this something I'm going to have to remember to do every time I reboot my VM?

Reply:
Shouldn't be - `sail artisan inertia:start-ssr` is meant for use in production environments.

`sail npm run dev` should be sufficient and is what I use for local dev (and also causes the console errors to not appear). I only see the console errors when Vite isn't running at all.

![Screenshot 2024-09-02 at 3 15 13 PM](https://github.com/user-attachments/assets/0fbd2acf-a8d8-460a-944b-92affe5780e6)

Note that I see this console output on _every_ page I navigate to, regardless of whether React is present:
```
[vite] connecting...
[vite] connected.
```
If you don't, something may be wrong with how the Vite dev server is spinning up locally in your environment.